Bricks/file/file.h

```diff
@@ -214,79 +214,125 @@ struct FileSystem {
   // TODO(dkorolev): Make OutputFile not as tightly coupled with std::ofstream as it is now.
   typedef std::ofstream OutputFile;
 
+  struct ScanDirItemInfo {
+    const std::string name;
+    const std::string path;
```

Comment:
On a second thought, `name` and `path` are shorter. I'm fine with either (as well as with `file` + `dir`, `filename` + `dirname`, etc.), and suggest @sompylasar jumps in :)